apps/web/app/future/settings/my-account/calendars/page.tsx

```diff
@@ -1,10 +1,41 @@
-import Page from "@pages/settings/my-account/calendars";
 import { _generateMetadata } from "app/_utils";
+import { getFixedT } from "app/_utils";
+import { getServerSession } from "next-auth";
+
+import { AUTH_OPTIONS } from "@calcom/feature-auth/lib/next-auth-options";
+import SettingsHeader from "@calcom/features/settings/appDir/SettingsHeader";
+import { Button } from "@calcom/ui";
+
+import CalendarsView from "~/settings/my-account/calendars-view";
 
 export const generateMetadata = async () =>
   await _generateMetadata(
     (t) => t("calendars"),
     (t) => t("calendars_description")
   );
 
+const Page = async () => {
+  const session = await getServerSession(AUTH_OPTIONS);
+  const t = await getFixedT(session?.user.locale || "en");
+
+  const AddCalendarButton = () => {
+    return (
+      <>
+        <Button color="secondary" StartIcon="plus" href="/apps/categories/calendar">
+          {t("add_calendar")}
+        </Button>
+      </>
+    );
+  };
+
+  return (
+    <SettingsHeader
+      title={t("calendars")}
+      description={t("calendars_description")}
+      CTA={<AddCalendarButton />}>
+      <CalendarsView />
```

Comment:
I think this should be `CalendarListContainer` instead? `CalendarsView` has the same CTA; e.g.,
```
const CalendarsView = () => {
  const { t } = useLocale();

  return (
    <>
      <Meta
        title={t("calendars")}
        description={t("calendars_description")}
        CTA={<AddCalendarButton />}
        borderInShellHeader={false}
      />
      <div className="mt-8">
        <CalendarListContainer />
      </div>
    </>
  );
};
```